get_limit_datacard.sh

```diff
@@ -21,12 +21,12 @@ get_last_cat() {
 
 pushd Datacard
   if [ $dy_bkg_model = 0 ]; then 
-    python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --n-in-sideband $indir/../CatOptim/N_in_sidebands.json --MH $mh --MX $mx --MY $my --procTemplate ${procTemplate} --prune --do-res-bkg --sig-syst $indir/../Interpolation/systematics.json
+    python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --n-in-sideband $indir/../CatOptim/N_in_sidebands.json --MH $mh --MX $mx --MY $my --procTemplate ${procTemplate} --prune --do-res-bkg --sig-syst systematics_all_merged.json
```

Comment:
This change was specific to your workflow, I don't think it should be committed.

Reply:
changed